src/TabbedView.js

```diff
@@ -54,7 +54,8 @@ class TabbedView extends Component {
         return;
       }
       this.renderedSceneKeys[key] = true;
-      scenes.push(this.renderScene(item, i));
+      const {navigationBarStyle,titleStyle} = this.props.navigationState;
+      scenes.push(this.renderScene({navigationBarStyle,titleStyle,...item}, i));
     });
     return (
       <View style={this.props.style}>
```

Comment:
Right button style and overall style for own nav bar passed to Router globally

Pavel.

> 5 июля 2016 г., в 17:44, Joe Noon <notifications@github.com> написал(а):
> 
> In src/TabbedView.js:
> 
> > @@ -54,7 +54,8 @@ class TabbedView extends Component {
> >          return;
> >        }
> >        this.renderedSceneKeys[key] = true;
> > -      scenes.push(this.renderScene(item, i));
> > +      const {navigationBarStyle,titleStyle} = this.props.navigationState;
> > +      scenes.push(this.renderScene({navigationBarStyle,titleStyle,...item}, i));
> >      });
> >      return (
> >        <View style={this.props.style}>
> @aksonov sorry about that! what did it break?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>